src/scenes/Wallet/components/EncryptDialog/index.js

```diff
@@ -71,7 +58,7 @@ export default class EncryptDialog extends Component {
                 type="password"
                 fullWidth
                 error={_.isEmpty(passphrase)}
-                onChange={(e) => this.setState({ passphrase: e.target.value })}
+                onChange={(e) => onPassphraseChange(e.target.value)}
```

Comment:
@alex-cory :
```js
const { wallet, wallet: {walletEncrypted} } = this.props.store;
onChange={(e) => wallet.passphrase = e.target.value}`;
```
change to this.

Reply:
```js

Reply:
better keep all the store change function inside the store file. Otherwise may cause confusion.